megamek/src/megamek/MegaMek.java

```diff
@@ -73,33 +62,30 @@
  * @author mev This is the class where the execution of the megamek game starts.
  */
 public class MegaMek {
+    private static MMLogger logger = null;
 
-    private static final MMLogger logger = DefaultMmLogger.getInstance();
-
-    public static String VERSION = "0.47.5-SNAPSHOT"; //$NON-NLS-1$
-    public static long TIMESTAMP = new File(PreferenceManager
-            .getClientPreferences().getLogDirectory()
-            + File.separator
-            + "timestamp").lastModified(); //$NON-NLS-1$
+    public static String VERSION = "0.47.5-SNAPSHOT";
+    public static long TIMESTAMP = new File(PreferenceManager.getClientPreferences().getLogDirectory()
+            + File.separator + "timestamp").lastModified();
 
     private static final NumberFormat commafy = NumberFormat.getInstance();
-    private static final String INCORRECT_ARGUMENTS_MESSAGE = "Incorrect arguments:"; //$NON-NLS-1$
-    private static final String ARGUMENTS_DESCRIPTION_MESSAGE = "Arguments syntax:\n\t MegaMek [-log <logfile>] [(-gui <guiname>)|(-dedicated)|(-validate)|(-export)|(-eqdb)|(-eqedb) (-oul)] [<args>]"; //$NON-NLS-1$
-    private static final String UNKNOWN_GUI_MESSAGE = "Unknown GUI:"; //$NON-NLS-1$
-    private static final String GUI_CLASS_NOT_FOUND_MESSAGE = "Couldn't find the GUI Class:"; //$NON-NLS-1$
-    private static final String DEFAULT_LOG_FILE_NAME = "megameklog.txt"; //$NON-NLS-1$
+    private static final String INCORRECT_ARGUMENTS_MESSAGE = "Incorrect arguments:";
+    private static final String ARGUMENTS_DESCRIPTION_MESSAGE = "Arguments syntax:\n\t MegaMek "
+            + "[-log <logfile>] [(-gui <guiname>)|(-dedicated)|(-validate)|(-export)|(-eqdb)|"
+            + "(-eqedb) (-oul)] [<args>]";
+    private static final String UNKNOWN_GUI_MESSAGE = "Unknown GUI:";
+    private static final String GUI_CLASS_NOT_FOUND_MESSAGE = "Couldn't find the GUI Class:";
+    private static final String DEFAULT_LOG_FILE_NAME = "megameklog.txt";
 
     public static void main(String[] args) {
-
         String logFileName = DEFAULT_LOG_FILE_NAME;
-
         CommandLineParser cp = new CommandLineParser(args);
 
         try {
             cp.parse();
             String lf = cp.getLogFilename();
             if (lf != null) {
-                if (lf.equals("none") || lf.equals("off")) { //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
Why are you removing this (here and elsewhere)?

Reply:
Visually distracting and increases the parsing time of a line of code, for something that is only used by a single IDE while running a specific setting.